### DISCUSSION TITLE
Changing a title of a link doesn't change the link text in linkDialogPlugin

### DISCUSSION THREAD

**Discussion by GydroCasper** [mdx-editor/editor#427] [General]
Hello,

If I change a title of a link via link editing dialog, it doesn't change the text in editor itself.

Reproduce:
1. Create a link with URL and Title.
2. Edit link and change the Title.
3. Submit
4. Text in editor shows old title.

![image](https://github.com/mdx-editor/editor/assets/19537952/505e1d83-6256-4b23-a623-22647858f176)
![image](https://github.com/mdx-editor/editor/assets/19537952/c99cb423-58e9-4934-9d05-3b0cb723c469)
![image](https://github.com/mdx-editor/editor/assets/19537952/42854649-5e6d-41b5-a499-f89f0a8ac61e)
![image](https://github.com/mdx-editor/editor/assets/19537952/c019e8ec-cece-4195-91b8-2778a68b8245)

**Comment by petyosi** [comment]
I can't reproduce this [on the live demo](https://mdxeditor.dev/editor/demo). Can you record a video?

**Reply by GydroCasper** [reply]
@petyosi Thank you for the quick response. Here is the video:

https://github.com/mdx-editor/editor/assets/19537952/82a5c43e-085a-4dc3-bcb2-62c58c8216bb

Any changes in Title don't affect the url text.

**Reply by petyosi** [reply]
Your video does not show the problem, and when I do that on my side, it works as expected, am I missing something?

**Reply by GydroCasper** [reply]
Ok, let's make a step back. What url title means in common? Do you mean, it is just an information field, that doesn't have any reference to the url text in the editor main window?
Our users understanding is that changing one leads to changing another. If that is not so, I'm a bit confused on the Title field purpose.
Could you tell me more about this, please.

**Reply by petyosi** [reply]
The title field changes the `title` attribute of the `a` HTML element. I am not sure what you mean by URL text.

**Reply by GydroCasper** [reply]
By url text I mean 
- Daring Fireball 
text in the screenshot.
![image](https://github.com/mdx-editor/editor/assets/19537952/02f8e740-d7dd-47e2-a1e5-0f213d36e478)

Yeah, now I understand, we have different meaning on Title (title html props that is used for tooltip and url value/text, shown in rich editor).
The confusing is that most users are not technically skilled and they don't understand, what HTML title property means. So, they expect that value to be corresponding to the url text they see.
Ok, I think, we can resolve it just by adding English translation for 'createLink.title' to rename it from Title to Tooltip.
Thank you.

**Reply by petyosi** [reply]
Got it. You can also do your own custom link dialog if you want to. With a bit of Lexical, you can also expose a field for the link text. Use the current LinkDialog component as a starting point.
